cranelift/filetests/filetests/runtests/simd-splat.clif

```diff
@@ -0,0 +1,33 @@
+test interpret
+test run
+target aarch64
+set enable_simd
+target x86_64
```

Comment:
We can fix the old backend errors by replacing the `target x86_64` with `target x86_64 machinst`.

This causes it to be ignored on the old style x86 backend.

Reply:
Perfect, let's do that then.